ServiceFinder/ServiceFinder.DAL/Repositories/AssistanceCategoryRepository.cs

```diff
@@ -24,5 +24,15 @@ public override async Task<PagedResult<AssistanceCategoryEntity>> GetAllAsync(in
             var query = Query.Include(ac => ac.Assistances);
             return await GetPagedResultAsync(query, pageNumber, pageSize, cancellationToken);
         }
+
+        public async override Task<AssistanceCategoryEntity> UpdateAsync(AssistanceCategoryEntity entity, CancellationToken cancellationToken)
+        {
+            var existingEntity = await Query.AsNoTracking()
```

Comment:
Remove the unused local variable 'existingEntity'. (https://rules.sonarsource.com/csharp/RSPEC-1481)